charts/cinder-csi-plugin/values.yaml

```diff
@@ -30,7 +30,7 @@ csi:
   plugin:
     image:
       repository: docker.io/k8scloudprovider/cinder-csi-plugin
```

Comment:
I guess we have to bump the chart version as well? and change the appversion here, https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/cinder-csi-plugin/Chart.yaml#L2

Reply:
https://github.com/kubernetes/cloud-provider-openstack/issues/1183 is the issue 
if we bump the version in this PR, then we can close that issue
otherwise I can submit a separate PR

Reply:
Yes the chart appVersion and image version needs to be changed to 1.19. Other than that, we should only make releases using named appVersion so I have added a branch filter in the github action to match pattern `release-*`.